src/client/proxy/handlers/storage.ts

```diff
@@ -8,21 +8,27 @@ import {Template, templateManager} from '../templateManager';
 // TODO: we don't handle multiple files upload. But if we want to,
 // we should change the response format
 const attachStorageHandlers = (server: FastifyInstance) => {
-    ['/_storage/', '/_storage'].forEach(route => {
-        server.post(route, async (req, res) => {
-            if (!req.headers['content-type']?.match('multipart/form-data')) {
-                return res.status(415).send('Only multipart/form-data is supported');
-            }
+    server.post('/_storage/', async (req: any, res: any) => {
+        if (!req.headers['content-type']?.match('multipart/form-data')) {
+            return res.status(415).send('Only multipart/form-data is supported');
+        }
 
-            const file = await req.file();
-            if (!file) {
-                return res.status(400).send('No files in the body');
+        const files = [];
+        const fields: Record<string, any> = {};
+        for await (const part of req.parts()) {
```

Comment:
This actually looks like a breaking change. I think `/_storage` path is used by our dapps and it is expected to accept files in the `file` field. @jensendarren could you please check that?

Reply:
The problem is that this endpoint is not working well and often freezes on file uploads, though this change doesn't help, so maybe it will be better to keep it as it was, yes